jsonapi-utils.gemspec

```
@@ -19,7 +19,7 @@ Gem::Specification.new do |spec|
   spec.executables   = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
   spec.require_paths = ['lib']
 
-  spec.add_runtime_dependency 'jsonapi-resources', '0.9.11'
+  spec.add_runtime_dependency 'jsonapi-resources', '0.10.4'
```

Any reason not to:

```suggestion
  spec.add_runtime_dependency 'jsonapi-resources', '~> 0.10.4'
```

---

Actually `jsonapi-utils` won't work with `0.10`. Lots of breaking changes between `0.9` and `0.10` so you want `~> 0.9.12` here. 

```suggestion
  spec.add_runtime_dependency 'jsonapi-resources', '~> 0.9.12'
```